src/pages/patientView/mutation/PatientViewMutationTable.tsx

```diff
@@ -50,13 +49,11 @@ export default class PatientViewMutationTable extends MutationTable<IPatientView
             MutationTableColumnType.CHROMOSOME,
             MutationTableColumnType.PROTEIN_CHANGE,
             MutationTableColumnType.MUTATION_TYPE,
-        ].concat(
```

Comment:
No need to conditionally define the column, `shouldExclude` property is designed to handle these kind of conditions.